src/test/java/restassured/endpoints/TestResourcesEndpoint.java

```diff
@@ -0,0 +1,73 @@
+package restassured.endpoints;
+
+import entity.Resource;
+import entity.ResourceListResponse;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.get;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+
+public class TestResourcesEndpoint {
+
+    private ResourceListResponse resourceListResponse;
+
+    @BeforeClass
+    public void setUp() {
+        RestAssured.baseURI = "http://localhost:4040";
+    }
+
+    @Test
+    public void testCorrectUserId() {
+        Response getUsers2Response = get("/api/unknown/2").then().extract().response();
+        getUsers2Response.then().statusCode(200).assertThat().body("data.name", equalTo("fuchsia rose"));
+    }
+
+    @Test
+    public void testIncorrectUserId() {
+        Response getUsers2Response = get("/api/unknown/23").then().extract().response();
```

Comment:
Is ```.then().extract().response()``` necessary? I believe ```get(...)``` will return a response.

Reply:
fixed.